src/plugins/explore/public/components/patterns_table/patterns_table_columns.tsx

```diff
@@ -3,14 +3,23 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { EuiBasicTableColumn } from '@elastic/eui';
+import { EuiBasicTableColumn, EuiButtonIcon } from '@elastic/eui';
 import { i18n } from '@osd/i18n';
 import React from 'react';
 import dompurify from 'dompurify';
 import { PatternItem } from './patterns_table';
 import { isValidFiniteNumber } from './utils/utils';
 
-export const patternsTableColumns: Array<EuiBasicTableColumn<PatternItem>> = [
+export const patternsTableColumns = (
+  openPatternsTableFlyout: (record: any) => void
+): Array<EuiBasicTableColumn<PatternItem>> => [
+  {
+    field: 'flyout',
+    width: '40px', // roughly size of the EuiButtonIcon
```

Comment:
Any way we can get this actual value?

Reply:
Will be leaving it with this value for now.